src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write your code here
+const allPopulation = document.querySelectorAll('.population');
+const allPopulationArray = Array.from(allPopulation);
+
+const allAveragePopulation = document.querySelectorAll('.average-population');
+const allAveragePopulationArray = Array.from(allAveragePopulation);
+
+const allTotalPopulation = document.querySelectorAll('.total-population');
```

Comment:
```suggestion
const total = document.querySelector('.total-population');
```

And you can remove `forEach `below and just update `total.textContent` value